fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
The issue is with https://jira.duraspace.org/browse/FCREPO-1628 and OSGi in general: the modeshape-jcr bundle depends on a certain cassandra driver which depends on an older version of guava (16.0). When bundles are resolved, I get a version dependency conflict (one dep chain needs v.16 and another dep chain needs v.18) and so I can't get the fcrepo-kernel-modeshape bundle to start up properly, and downgrading to guava 16 for fcrepo isn't an option.